src/main/java/seedu/address/ui/InternshipCard.java

```diff
@@ -52,6 +54,11 @@ public InternshipCard(Internship internship, int displayedIndex) {
         } else {
             contactPersonId.setText(internship.getContactPersonId().id.toString());
         }
+        if (internship.getInterviewDate() == null) {
+            interviewDate.setText("No interviews scheduled.");
+        } else {
+            interviewDate.setText(internship.getInterviewDate().toString());
```

Comment:
Would it be be better to have this in the GUI?
```suggestion
            interviewDate.setText("Interview on: " + internship.getInterviewDate().toString());
```